client/src/modules/stock/entry/entry.js

```diff
@@ -331,7 +333,7 @@ function StockEntryController(
       from_depot : vm.movement.entity.instance.depot_uuid,
       to_depot : vm.depot.uuid,
       document_uuid : vm.movement.entity.instance.document_uuid,
-      date : vm.movement.entity.instance.date,
+      date : vm.movement.date,
```

Comment:
:dancer:

Reply:
Are you sure this bug doesn't affect other modules (exit)?  Or other entry types?